### PR TITLE
chore(args): make compatible severity and services arguments

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Make compatible severity and services arguments.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/lib/cli/parser.py [hunk 1]
```diff
@@ -220,7 +220,7 @@ def __init_checks_parser__(self):
         group.add_argument(
             "-s", "--services", nargs="+", help="List of services to be executed."
         )
-        group.add_argument(
+        common_checks_parser.add_argument(
```
Comment (jfagoagas) [inline prowler/lib/cli/parser.py:223]: Please include a test in the `tests/lib/cli/parser_test.py` class to verify the use of `--services` with `--severities`.

**Review by jfagoagas — CHANGES_REQUESTED** [review]
Fast and easy!! Good one @sergargar 

Please include the test I suggested and then we'll merge this 🚀

File: prowler/lib/cli/parser.py (hunk 1 above)
Comment (MrCloudSec) [reply prowler/lib/cli/parser.py:223]: Done!

**Review by jfagoagas — APPROVED** [review]
Great !!

**Pull request merged**